packages/ui/src/components/SearchField/types.ts

```diff
@@ -38,4 +38,9 @@ export interface SearchFieldProps
    * The placeholder text for the input
    */
   placeholder?: string;
+
+  /**
+   * The icon to render in the input
+   */
+  startCollapsed?: boolean;
```

Comment:
Hmm, is this support to be `collapsible` instead? The behavior doesn't seem to match the naming

Reply:
Sorry I did that quickly but I changed it to be startedCollapsed instead and forgot to update it everywhere.

Reply:
This is now fixed.